galaxy/api/serializers/serializers.py

```diff
@@ -363,7 +374,7 @@ def get_related(self, obj):
         return res
 
     def get_summary_fields(self, obj):
-        if obj is None:
+        if obj is None or getattr(obj, 'repository') is None:
```

Comment:
Why are there no summary fields for collection importing?

Reply:
These edits are to get collection imports storing import task messages, `api/v1/imports` edits are temporary (right now to verify task logging) until `api/v2/collection` views are in place